install-project.sh

```diff
@@ -1,16 +1,23 @@
-
 slug=$1
-MVNOPTIONS=$2
+MVNOPTIONS="$2 -Dmaven.repo.local=$AZ_BATCH_TASK_WORKING_DIR/input/dependencies"
 USER=$3
 module=$4
 sha=$5
 dir=$6
 fullTestName=$7
 RESULTSDIR=$8
+projectname=${slug%/*}
```

Comment:
We should be able to delete this variable and just directly use ${slug%/*} in Line 154